client/me/privacy/main.jsx

```diff
@@ -96,7 +96,7 @@ const Privacy = createReactClass( {
 							</p>
 							<p>
 								{ translate(
-									'We use other tracking tools, including some from third parties. ' +
+									'In addition to our analytics tool, we use other tracking tools, including some from third parties. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'We use other tracking tools, including some from third parties. {{cookiePolicyLink}}Read about these{{/cookiePolicyLink}} and how to control them.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).